pub-critic/src/app/components/sideMenu/SideMenu.module.scss

```diff
@@ -0,0 +1,68 @@
+.SideMenu {
+    width: 100%;
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    display: flex;
+    flex-direction: row;
+}
+.Backdrop {
+    width: 70%;
+    height: 100%;
+    background-color: black;
+    transform: translateX(-300%);
```

Comment:
why 300%?

Reply:
slight misinput in code, fixed :white_check_mark: